aries-site/src/examples/components/checkbox/CheckBoxDisabledExample.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Box, CheckBox, FormField } from 'grommet';
+
+export const CheckBoxDisabledExample = () => {
+  return (
+    <Box>
+      <FormField label="Label" htmlFor="disabled-checkbox" disabled>
+        <CheckBox
+          label="disabled"
+          checked
+          disabled
+        />
+      </FormField>
+      <FormField label="Label" htmlFor="disabled-checkbox" disabled>
```

Comment:
Think the htmlFor needs an `id` on the Checkbox with the same value to link them in the DOM -- also, I think the values need to be unique.

Suggestion:
For the first one, have `htmlFor="disabled-checkbox"` and on the CheckBox add `id="disabled-checkbox"`.
For the toggle, follow the same structure but call it `disabled-toggle`